Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/service/PersistedTasksService.java

```diff
@@ -51,20 +56,15 @@ public PersistedTasksService(
     }
 
     @Override
-    public Observable<Event<Tasks>> getTasksEvents() {
+    public Observable<Event<Tasks>> getTasksEvent() {
```

Comment:
Singular, `getTask(Id)` for example, you get the Task and the subsequent updates to it.